gbpservice/nfp/base_configurator/api/v1/controllers/controller.py

```diff
@@ -108,7 +91,7 @@ def get(self):
         try:
             if not cache_ips:
                 notification_data = jsonutils.dumps(notifications)
-                msg = ("NOTIFICATION_DATA sent to config_agent %s"
+                msg = ("NOTIFICATION_DATA sent to config_orchestrator %s"
```

Comment:
The notifications are not sent to config orchestrator. Change this log and say "Notification sent: <data dump>"